src/executables/util_cs.c

```diff
@@ -3275,51 +3280,60 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	    {
 	      goto check_applied_info_end;
 	    }
-	  error = db_restart (arg->command_name, TRUE, local_database_name);
-	  if (error != NO_ERROR)
+	  if ((error = db_restart (arg->command_name, TRUE, local_database_name)) != NO_ERROR)
 	    {
 	      goto check_applied_info_end;
 	    }
 
 	  if (HA_DISABLED ())
 	    {
+	      check_applied_info = false;
+	      printf ("\nERROR : ");
 	      PRINT_AND_LOG_ERR_MSG (msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_APPLYINFO,
 						     APPLYINFO_MSG_NOT_HA_MODE));
 	      goto check_applied_info_end;
 	    }
-
-	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
-				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
-				     &applied_final_lsa);
-	  (void) db_shutdown ();
-	}
-      else if (check_copied_info)
-	{
-	  memset (local_database_name, 0x00, CUB_MAXHOSTNAMELEN);
-	  strcpy (local_database_name, database_name);
-	  strcat (local_database_name, "@localhost");
-
-	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
-				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
-				     &applied_final_lsa);
+	  error =
+	    la_get_applyinfo_applied_log_info (database_name, log_path, check_replica_info, verbose,
+					       &applied_final_lsa);
+	  if (error != NO_ERROR)
+	    {
+	      check_applied_info = false;
+	      error = NO_ERROR;
```

Comment:
```suggestion
	      goto check_applied_info_end;
```

Reply:
I think you can remove 'PRINT_AND_LOG_ERR_MSG ("ERROR : %s\n", db_error_string (3));' in the la_get_applyinfo_applied_log_info function and then you can apply the previous recommand by me.

Reply:
```suggestion
	      check_applied_info = false;
	      error = NO_ERROR;
```
I think this suggestion does not seem so applied to PR, because `la_get_applyinfo_applied_log_info()` function prints error message inside its function. So, the return value of this function just help to whether `Delay in Applying Copied Log` is printed.